src/sheetFunctions.js

```diff
@@ -428,3 +428,22 @@ function SAN_TOP_HOLDERS_PERCENT_OF_TOTAL_SUPPLY (projectSlug, from, to, numberO
 function SAN_HISTORICAL_BALANCE_DEDUP (projectSlug, from, to, address) {
   return handleErrors_(historicalBalanceDedup_)(projectSlug, from, to, address)
 }
+
+/**
+ * Returns the reversed array of the results
+ * @param {Array} array The array of results
+ * @returns {Array} of reversed results
+ * @customfunction
+ */
+function SAN_REVERSE (array) {
+  [headers, ...rest] = array
+  array.shift()
+
+  if (!Array.isArray(array) || array.length === 0) {
+    return null
+  }
```

Comment:
These checks should be placed before the operations above

Reply:
Actually, the `array.shift()` is done twice - above and below. There's some code that needs to be deleted.

Reply:
ah yes, I've somehow missed that. Thanks :ok_hand: